src/components/AppBar/NavBar.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import Logo from 'assets/images/logo';
+import Hidden from '@material-ui/core/Hidden';
+import MenuIcon from '@material-ui/icons/Menu';
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@material-ui/core/IconButton';
+import ArrowBack from '@material-ui/icons/ArrowBack';
+import * as Style from './NavBar.style';
+import MobileMenu from './MobileMenu';
+import { DonateButtonWithoutFade, DonateButtonWithFade } from './DonateButton';
+
+const isLocationPage = (pathname: string) => pathname.includes('/us');
+
+const isHomePage = (pathname: string) =>
+  ['/', '/alert_signup', '/compare'].includes(pathname);
+
+const NavBar: React.FC = () => {
+  const [isMenuOpen, setMenuOpen] = useState(false);
+  const closeMenu = () => setMenuOpen(false);
+
+  // We only fade the donate button on the home page on mobile, where the donate
+  // button doesn't appear until the banner is scrolled away.
+  const { pathname } = useLocation();
+  const MobileDonateButton = isHomePage(pathname)
+    ? DonateButtonWithFade
+    : DonateButtonWithoutFade;
+
+  return (
+    <Style.AppBar position="sticky" color="transparent" elevation={0}>
+      <Style.Toolbar>
+        <Style.BackLink to="/">
+          {isLocationPage(pathname) && <ArrowBack />}
+        </Style.BackLink>
+        <Link to="/">
+          <Logo />
+        </Link>
+        <Style.Spacer />
+        <Hidden smDown>
+          <Style.NavLink
+            to="/"
+            key="map"
+            activeClassName="active"
+            isActive={(match, { pathname }) => isHomePage(pathname)}
+          >
+            Map
+          </Style.NavLink>
+          <Style.NavLink to="/about" key="about" activeClassName="active">
+            About
+          </Style.NavLink>
+          <Style.NavLink
+            to="/resources"
+            key="resources"
+            activeClassName="active"
+          >
+            Resources
+          </Style.NavLink>
+          <Style.TabLink href="https://blog.covidactnow.org" key="blog">
+            Blog
+          </Style.TabLink>
+          <Style.NavLink to="/contact" key="contact" activeClassName="active">
+            Contact Us
+          </Style.NavLink>
+          <DonateButtonWithoutFade />
+        </Hidden>
+        <Hidden mdUp>
```

Comment:
this breakpoint needs fixing (it should transition from mobile->desktop at 800px). this MUI breakpoint is 960px, but the `StyledMobileMenu` is set to `display: none` until 800px, so there is no navbar showing up between 800-960px:

![Screen Shot 2020-10-20 at 1 34 46 PM](https://user-images.githubusercontent.com/44076375/96623992-25b93f80-12da-11eb-8f01-83b0a063ee5f.png)

Reply:
Good catch! I removed the media query since the mobile menu and button don't render for screens sized `md` and up, it works correctly now 👍